src/backend/updates.rs

```diff
@@ -41,7 +42,13 @@ fn init() {
 
 #[pre_upgrade]
 fn pre_upgrade() {
-    mutate(env::memory::heap_to_stable)
+    HEAP.with(|cell| {
```

Comment:
How many instructions does this take with the production state?

Reply:
Will check. But since this will triggered only after a successful migration, we would have no posts in the heap, and the heap size will be around 23MB.

When the migration fails for some reason and all posts stayed in the heap, I can't see many options without deleting posts, can you?